addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15592,7 +15592,13 @@ msgctxt "#24136"
 msgid "Would you like to enable this add-on?"
 msgstr ""
 
-#empty strings from id 24137 to 24138
+#. Button in Add-on Manager -> Available Updates for updating all possible add-ons when auto-updating is off.
+#: xbmc/filesystem/AddonDirectory.cpp
+msgctxt "#24137"
+msgid "Update all auto-updateable add-ons"
```

Comment:
And for this one `Install auto updates only`.

I think that is about as concise as we can get it.